src/StoredEvents/Models/EloquentStoredEventQueryBuilder.php

```diff
@@ -32,7 +34,10 @@ public function whereAggregateRoot(string $uuid): self
 
     public function whereEvent(string ...$eventClasses): self
     {
-        $this->whereIn('event_class', $eventClasses);
+        $this->whereIn('event_class', array_map(
+            static fn (string $eventClass): string => StoredEvent::getEventClass($eventClass),
```

Comment:
```suggestion
           fn (string $eventClass): string => StoredEvent::getEventClass($eventClass),
```

I assume for performance or an IDE default? But it's negligible in this case, would rather have it without.